apps/modernization-ui/src/apps/page-builder/pages/ConditionLibrary/ConditionLibraryTable.tsx

```diff
@@ -0,0 +1,149 @@
+    { name: Column.ConditionFamily, sortable: true },
+    { name: Column.CoinfectionGroup, sortable: false },
+    { name: Column.NND, sortable: true },
+    { name: Column.InvestigationPage, sortable: true },
+    { name: Column.Status, sortable: true }
+];
+
+const ConditionLibraryTable = ({ conditions, currentPage, pageSize, totalElements }: ConditionLibraryTableProps) => {
+    const [tableRows, setTableRows] = useState<TableBody[]>([]);
+    const { setCurrentPage, setSortBy, setSortDirection, searchQuery, setSearchQuery } = useContext(ConditionsContext);
+
+    /*
+     * Converts header and Direction to API compatible sort string such as "name,asc"
+     */
+    const toSortString = (name: string) => {
+        if (name) {
+            switch (name) {
+                case Column.Condition:
+                    setSortBy(`conditionShortNm`);
+                    break;
+                case Column.Code:
+                    setSortBy(`id`);
+                    break;
+                case Column.ProgramArea:
+                    setSortBy(`progAreaCd`);
+                    break;
+                case Column.ConditionFamily:
+                    setSortBy(`familyCd`);
+                    break;
+                case Column.NND:
+                    setSortBy(`nndInd`);
+                    break;
+                case Column.InvestigationPage:
+                    setSortBy(`investigationFormCd`);
+                    break;
+                case Column.Status:
+                    setSortBy(`statusCd`);
+                    break;
+                default:
+                    setSortBy('conditionShortNm');
+                    break;
+            }
+        }
+    };
+
+    const handleSort = (name: string, direction: Direction): void => {
+        if (currentPage > 1 && setCurrentPage) {
+            setCurrentPage(1);
+        }
+
+        if (direction === Direction.None) {
+            toSortString(Column.Condition);
+            setSortDirection(Direction.Ascending);
+        } else {
+            toSortString(name);
+            setSortDirection(direction);
+        }
+    };
+
+    const asTableRow = (condition: Condition): TableBody => ({
+        id: condition.id,
+        expanded: false,
+        tableDetails: [
+            {
+                id: 1,
+                title: (
+                    <div className="page-name">
+                        <Link to={`/page-builder/edit/condition/${condition.id}`}>{condition.conditionShortNm}</Link>
+                    </div>
+                )
+            },
+            { id: 2, title: condition.id },
+            { id: 3, title: condition.progAreaCd },
+            {
+                id: 4,
+                title: condition.familyCd
+            },
+            { id: 5, title: condition.coinfectionGrpCd },
+
+            { id: 6, title: condition.nndInd },
+            { id: 7, title: condition.investigationFormCd },
+            { id: 8, title: condition.statusCd }
```

Comment:
The status values are showing as `I` or `A`, should these be `Active` or `Inactive`? 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/f1c01438-c7ac-4319-8e79-bb982de6b971)